docs/_documentations/remote-codewind-overview.md

```diff
@@ -10,33 +10,33 @@ type: document
 
 # Overview: Using Codewind remotely
 
-Codewind can be used in one of three ways - [locally](./gettingstarted.html), [hosted](./che-installinfo.html) as an application on the cloud, or remotely. 
+Codewind can be used in one of three ways: [locally](./gettingstarted.html), [hosted](./che-installinfo.html) as an application on the cloud, or remotely. 
 
-By **deploying and using Codewind remotely**, you can **develop your code locally** using your local Codewind installation, but **build and run your application in the cloud**. Remote use of Codewind frees up desktop resources, using the cloud's resources to build and run applications. 
+Deploy and use Codewind remotely to develop your code locally but build and run your application in the cloud. Use the cloud's resources to build and run applications and free desktop resources. 
 
-Follow the instructions to get started with **using Codewind remotely**. This will guide you through: 
+Follow the instructions to get started with using Codewind remotely:
 
 1. Configuring your local IDE's Codewind extension to connect to Codewind in the cloud
 2. Registering all necessary security parameters
-3. Creating a new project that builds and runs in the cloud
+3. Creating a project that builds and runs in the cloud
 4. Building and running an existing project in the cloud
 
-**Note: Deploying Codewind remotely** Before using Codewind remotely, Codewind must be deployed to the cloud. If you do not have a remote deployment of Codewind you must follow instructions to [deploy Codewind remotely](./remote-deploying-codewind.html) before [connecting your local Codewind to the cloud](./remote-codewind-overview.html).
+**Note:** Before you use Codewind remotely, Codewind must be deployed to the cloud. If you do not have a remote deployment of Codewind, follow instructions to [deploy Codewind remotely](./remote-deploying-codewind.html) before you [connect your local Codewind to the cloud](./remote-codewind-overview.html).
 
 ### Prerequisites
 
-Before using Codewind remotely, you must:
+Before you use Codewind remotely, complete these steps:
 
-1. **Install Codewind into your preferred IDE on your local machine.** 
+1. **Install Codewind into your preferred IDE on your local computer.** 
```

Comment:
Rob wanted us to use the terminology local machine. (FYI We have an oustanding task to update the Acrolinx config files with the terminology specific to Codewind).